src/overlays/actors/ovl_En_Hgo/z_en_hgo.c

```diff
@@ -196,109 +207,111 @@ void EnHgo_DefaultDialogueHandler(EnHgo* this, PlayState* play) {
     this->actor.shape.rot.y = this->actor.world.rot.y;
 }
 
-void func_80BD06FC(EnHgo* this, PlayState* play) {
+// Handles the next dialogue based on player's selection
+void EnHgo_HandlePlayerChoice(EnHgo* this, PlayState* play) {
     if (Message_ShouldAdvance(play)) {
-        switch (this->unk_314) {
+        switch (this->textId) {
             case 0x158F:
                 Message_StartTextbox(play, 0x1590, &this->actor);
-                this->unk_314 = 0x1590;
+                this->textId = 0x1590;
                 break;
             case 0x1590:
```

Comment:
```suggestion
                break;
                
            case 0x1590:
```
Same here, newlines please :)